src/DbfDataReader/DbfDataReader.cs

```diff
@@ -20,6 +21,20 @@ public DbfDataReader(string path, DbfDataReaderOptions options)
             DbfRecord = new DbfRecord(DbfTable);
         }
 
+        public DbfDataReader(Stream stream, DbfDataReaderOptions options)
+        {
+            _options = options;
+            DbfTable = new DbfTable(stream, options.Encoding);
+            DbfRecord = new DbfRecord(DbfTable);
+        }
+
+        public DbfDataReader(Stream stream, Stream memoStream, DbfDataReaderOptions options)
```

Comment:
My biggest concern is that these two streams are handled differently. DbfTable's BinaryReader is configured to leave the stream open when it is closed/disposed. DbfMemo takes the default behavior of closing the stream.

I would personally be fine only exposing the 1st constructor which doesn't accept a memo stream.

Reply:
In the DbfMemo stream constructor, the BinaryReader is opened without a 3rd argument of `true`, which tells BinaryReader to leave the underlying stream open when the reader is closed or disposed. In DbfTable, however, the 3rd argument *is* supplied. The impact is for the existing DbfTable constructor which accepts a memoStream parameter, the primary stream and the memoStream have different behavior (memoStream will be closed when the chain is disposed, but main stream will not).

While it seems like it would be ideal to have these behave in the same way (I favor leaving the stream open since the objects don't "own" it), it could be a breaking change for any current users who have coded against the current behavior of the public DbfTable(stream, stream, encoding) constructor, so I didn't update the existing behavior. I don't _need_ the memoStream for my update, so I would also be okay just removing the DbfDataReader constructor which accepts two streams, in order to not propagate the "problem" further for now.

Reply:
Ah yes, good spot. I'll merge your changes in and bump the version to 0.5.2. And then update to make the stream behaviour consistent and bump the version to 0.6.